src/dsymbol.d

```diff
@@ -1207,6 +1207,9 @@ private:
     Dsymbols* importedScopes;
     PROTKIND* prots;            // array of PROTKIND, one for each import
 
+    import std.bitmanip : BitArray;
+    BitArray accessiblePackages;// whitelist of accessible (imported) packages
```

Comment:
We're already completely relying on druntime, why would using BitArray be any additional liability?
Same goes for std.algorithm.partition or binary search. There is no reason why those should ever change.